pulsar-broker/src/main/java/org/apache/pulsar/broker/service/AbstractReplicator.java

```diff
@@ -197,6 +198,7 @@ public void startProducer() {
             ProducerBuilderImpl builderImpl = (ProducerBuilderImpl) producerBuilder;
             builderImpl.getConf().setNonPartitionedTopicExpected(true);
             builderImpl.getConf().setReplProducer(true);
+            builderImpl.getConf().setAccessMode(ProducerAccessMode.Shared);
```

Comment:
Shared is the default mode, so this change doesn't seem to be necessary.
https://github.com/apache/pulsar/blob/39bb67542f2a7b849acaff681d408c693e1a2a18/pulsar-client/src/main/java/org/apache/pulsar/client/impl/conf/ProducerConfigurationData.java#L207